### PR TITLE
Add Citizen Science

### PR DESCRIPTION
**Pull request by dylanrees**
https://github.com/dylanrees/toolsforcitizenscience

This list contains a large collection of tools for citizen scientists.  I couldn't find a list similar to it, especially on GitHub; hence my desire to create it.  It's got a open-source software section, which GitHub users in particular should be useful in building, but it also lists resources such as publications, open-source equipment suppliers, physical spaces that are practicing citizen science, and a few other things.

# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!

- [X] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [X] This pull request has a descriptive title. *(For example: `Add Node.js`)*
- The list I added
	- [x] **has been around for at least 20 days,**
	- [x] is a non-generated Markdown file in a GitHub repo,
	- [x] is not a duplicate,
	- [x] only has awesome items with descriptions *(Awesome lists are curations of the best, not of everything)*,
	- [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] has a Table of Contents section named `Contents` as the first section,
	- [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)*,
	- [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [x] has been checked for inconsistent formatting, spelling, and grammar *(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)*,
	- [x] has been added at the bottom of the appropriate category.

### CODE REVIEW

**Comment by aleksandar-todorovic** [comment]
You've checked the second to last part, even though this part is not respected at all:

> This includes starting each link description with an uppercase character and ending it with a period.

Also, it is advisable to put the contributing guidelines in a `contributing.md` file, because once you do so, everyone who tries to create pull request will see the banner at the top asking the submitter to read them before submitting.

I would also strongly recommend putting the entire content of CC0 into a `LICENSE.md` file.

**Comment by dylanrees** [comment]
Thanks for the suggestions.  I went ahead and implemented them.

**Comment by sindresorhus** [comment]
Cool idea! :)

Some nitpick:

- The repo name could use some dashes. `toolsforcitizenscience` => `tools-for-citizen-science`. Right now it's totally unreadable. Or even better: `citizen-science-tools`. Actually, why have `tools` in the name at all? `Citizen Science` and `citizen-science` would read much better.
- `Table of Contents` => `Contents` And the table should use `-` for the list, not numbers.
- Drop `# The List`
- Section headings should be properly capitialized and without `:`
- `github` => `GitHub`

**Comment by dylanrees** [comment]
Good suggestions!  I've implemented them all.

**Comment by sindresorhus** [comment]
Any comment on just naming it `Citizen Science`?

**Comment by dylanrees** [comment]
At first I wanted to include "tools" for fear of being too broad.  However, it seems like this is the only list project of its type on GitHub so actually I think citizen-science is good.  I changed the name.

File: readme.md [hunk 1]
```diff
@@ -418,6 +418,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Ripple](https://github.com/vhpoet/awesome-ripple) - Open source distributed settlement network.
 - [Katas](https://github.com/gmontalvoriv/awesome-katas)
 - [Tools for Activism](https://github.com/drewrwilson/toolsforactivism)
+- [Tools for Citizen Science](https://github.com/dylanrees/toolsforcitizenscience)
```
Comment (sindresorhus) [inline readme.md]: Can you update this accordingly? And add a very short description.

**Comment by dylanrees** [comment]
Done!

**Pull request merged**